### DISCUSSION TITLE
Can't install Kali 2022 arm on UTM

### DISCUSSION THREAD

**Discussion by gitblanc** [utmapp/UTM#4461] [General]
I'm trying to install kali, but never gets me the install menu. It just stays there forever. Don't really know what happens with it. I hope you can help me :)

**Comment by DylanChuckry** [comment]
The graphic drivers do not install by default but it's no issue. Set the display type to "console only" and it will let you install. Once it has installed, shut the machine down, eject the disk and set the display type back to "full graphics."

**Reply by gitblanc** [reply]
I'll try it and comment if it works.
Otherwise thank you a lot :)

**Reply by gitblanc** [reply]
I tried it and i can install it, but i can't run it xd. Is there a way or command to do it? Or do I need something else?
<img width="573" alt="imagen" src="https://user-images.githubusercontent.com/87705461/193092945-161332f5-85c6-43a4-9599-534f46b1f738.png">

**Reply by DylanChuckry** [reply]
If you can install it you can run it, you just have to switch back to full graphics.

**Reply by gitblanc** [reply]
But it still doesn't load :/. Don't really know what happens
